### PR TITLE
Add code of conduct #09

### PR DESCRIPTION
**Pull request by thuthu-c**
Fixed issue https://github.com/UFRN-URNAI/urnai-tools/issues/9.

The code of conduct s adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 2.1.

### CODE REVIEW

**Review by alvarofpp — CHANGES_REQUESTED** [review]
Fixes: 
 
- It is necessary that you follow the pattern of Conventional Commits in your commits. Example: `docs: Add code of conduct`.

Previous PRs:

- #52
- #60
- #61

**Comment by alvarofpp** [comment]
@thuthu-c, please follow good software development practices and don't create multiple PRs on the same change. This generates a lot of unnecessary logging.

Reuse that same PR from now on for the development task.

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (alvarofpp) [inline .github/CODE_OF_CONDUCT.md]:
```suggestion
the repository's discussion tab, in pull requests or issues, or through direct contact with the repository's maintainers.
```

What do you think?

**Review by alvarofpp — CHANGES_REQUESTED** [review]

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (thuthu-c) [reply .github/CODE_OF_CONDUCT.md]: I had a doubt about this, but I'll solve it with the right guidelines!

Comment (thuthu-c) [reply .github/CODE_OF_CONDUCT.md]: Solved this!!!

**Review by alvarofpp — APPROVED** [review]

**Pull request merged**